src/main/scala/histogramactors/HistogramMain.scala

```diff
@@ -0,0 +1,137 @@
+package histogramactors
+
+import akka.actor.{Actor, ActorRef, ActorSystem, Props}
+import akka.pattern.{ask, pipe}
+import akka.util.Timeout
+
+import java.io.PrintWriter
+import scala.collection.mutable
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.concurrent.duration.{Duration, MILLISECONDS}
+import scala.io.Source
+import scala.util.{Failure, Success, Try, Using}
+
+object HistogramMain extends App {
+  val inputFilePath = "src/main/resources/histogram/input.txt"
+  val outputFilePath = "src/main/resources/histogram/output.txt"
+
+  val system = ActorSystem("histogram")
+  val master = system.actorOf(Props(classOf[Master], inputFilePath, outputFilePath, system), "master")
```

Comment:
You don't need to pass the system to an actor. You could always get it from the actor context.